examples/src/main/scala/org/apache/carbondata/examples/CarbonExample.scala

```diff
@@ -32,24 +32,34 @@ object CarbonExample {
 
     cc.sql("DROP TABLE IF EXISTS t3")
 
+    // create table, 6 dimensions, 1 measure
     cc.sql("""
            CREATE TABLE IF NOT EXISTS t3
            (ID Int, date Timestamp, country String,
            name String, phonetype String, serialname String, salary Int)
            STORED BY 'carbondata'
            """)
 
+    // load data using kettle
     cc.sql(s"""
            LOAD DATA LOCAL INPATH '$testData' into table t3
            """)
 
+    // load data without kettle
```

Comment:
suggest either create a different example for without kettle or do the different query respectively for two type of load data.